### DISCUSSION TITLE
k8s 内ilogtail资源异常问题

### DISCUSSION THREAD

**Discussion by wkai886** [alibaba/ilogtail#503] [Help]
以 daemonset 方式部署，偶尔会有一个 pod，cpu 和 内存 占用都会飙到 100%， 此时进程卡死，无法 exec -it 进入 pod，日志收集也停止，重建 pod 后会恢复 或者 一段时间后 被oom kill 后也会恢复。 资源 limit 8c  4g  request  2c 1g。

cpu 占用：
![image](https://user-images.githubusercontent.com/41099009/199450122-66646a89-7553-4976-9f3a-dc98fda416a1.png)

内存占用:
![image](https://user-images.githubusercontent.com/41099009/199450248-46a315a2-d2dc-48c9-911a-ae4a1687918b.png)

看pod 相关日志暂未发现啥异常

**Comment by messixukejia** [comment]
可以看看日志有什么打印
